Microsoft.Toolkit.Uwp.UI.Controls/GridSplitter/GridSplitter.Events.cs

```diff
@@ -60,23 +54,32 @@ private void GridSplitter_Loaded(object sender, RoutedEventArgs e)
             _hoverWrapper = hoverWrapper;
         }
 
-        private void Gripper_KeyDown(object sender, KeyRoutedEventArgs e)
+        private void CreateGripperDisplay()
         {
-            var gripper = sender as GridSplitterGripper;
-
-            if (gripper == null)
+            if (_gripperDisplay == null)
             {
-                return;
+                var gripperText = _resizeDirection == GridResizeDirection.Columns ? GripperBarVertical : GripperBarHorizontal;
+                _gripperDisplay = new TextBlock
+                {
+                    FontFamily = new FontFamily(GripperDisplayFont),
+                    HorizontalAlignment = HorizontalAlignment.Center,
+                    VerticalAlignment = VerticalAlignment.Center,
+                    Foreground = GripperForeground,
+                    Text = gripperText
```

Comment:
I see this variable can be replace with it's value directly, no need to create a new variable.